components/button/button-icon.js

```diff
@@ -126,9 +128,9 @@ class ButtonIcon extends ButtonMixin(VisibleOnAncestorMixin(RtlMixin(LitElement)
 				?formnovalidate="${this.formnovalidate}"
 				formtarget="${ifDefined(this.formtarget)}"
 				name="${ifDefined(this.name)}"
-				title="${ifDefined(this.text)}"
 				type="${this._getType()}">
 				<d2l-icon icon="${ifDefined(this.icon)}" class="d2l-button-icon"></d2l-icon>
+				${this.text ? html`<d2l-tooltip for="target" disable-focus-lock>${this.text}</d2l-tooltip>` : null}
```

Comment:
I added `disable-focus-lock` because a common pattern in the LMS involves having multiple `d2l-button-icon`s in a row, all with tooltips. `disable-focus-lock` means the tooltip will close when the user mouses off so they don't get visibly overlapped tooltips.